packages/server/src/file/file.resolver.ts

```diff
@@ -0,0 +1,34 @@
+import { Args, Mutation, Parent, Query, ResolveField, Resolver } from '@nestjs/graphql';
+import { FileService } from './file.service';
+import { File } from './file.model';
+import { CreateFileInput } from './file.dto';
+import { UseGuards } from '@nestjs/common';
+import { JwtAuthGuard } from 'src/auth/jwt.guard';
+import { Comment } from 'src/comment/comment.model';
+import { CommentService } from 'src/comment/comment.service';
+
+@Resolver(() => File)
+@UseGuards(JwtAuthGuard)
+export class FileResolver {
+  constructor(private fileService: FileService, private commentService: CommentService) {}
+
+  @Query(() => File)
```

Comment:
Need to mark this as nullable, otherwise an error is thrown if a file is not returned